how-to/privileged.md

```diff
@@ -0,0 +1,45 @@
+# Privileged Kata Containers
+
+Kata Containers supports creation of containers that are "privileged" - i.e. have additional capabilities and access
+that isn't normally granted.
+
+* [Caveats](#caveats)
+    * [Host Devices](#host-devices)
+        * [Containerd and CRI](#containerd-and-cri)
+
+## Caveats
```

Comment:
Although I personally like this word, I wonder if "Warnings" would be clearer for non-native speakers?